### ISSUE TITLE
Improve avatar support, cache locally.

### ISSUE THREAD

**Issue by kickscondor** [kickscondor/fraidycat#7]
I just haven't put in the work to polish this up. Sometimes favicons aren't appearing. Sometimes they're just not loading.

**Comment by kickscondor** [comment]
So, here's what the rules should probably by closer to - and this coincides with what's needed for bundles.

* Start with the icon URL from Fraidyscrape (the feed icon usually).
* Check for its existence and that it's an image.
  * If so, cache it.
* Check for a "shortcut icon" or "og:image" or something like it on the home page (if one exists).
  * Works? Keep it.
* Lastly, check for a valid favicon.ico.
  * If found, cache that.
* Nvm. Generic globe or whatever.

**Comment by kickscondor** [comment]
Also 'Strict Tracking' is causing avatars on (say) pbs.twimg.com to get blocked - and I can't seem to whitelist them - so this is a more urgent item.

**Comment by kickscondor** [comment]
Yeah - and in my testing - the problem here is that the extension has whitelisted the domain, but https://fraidyc.at/s/ doesn't have access to it.

**Comment by Brin-o** [comment]
Not sure if things are still being worked on, but since a lot of pages (twitter, youtube) where you can track people simply use the site favicon, a simple fix would be to allow the person to pick an icon/emoji/pic to be used as the favicon.

More complex obviously, to grab the avatar of the profile.

**Comment by kickscondor** [comment]
@Brin-o I would like to do that eventually - is just a lower priority at the moment. Thank you!
